### PR TITLE
Adapt to recent PRs

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Autoupgrade PR for recent PRs - https://github.com/PrestaShop/PrestaShop/pull/37236/ and https://github.com/PrestaShop/PrestaShop/pull/36873/ and added a comment about https://github.com/PrestaShop/PrestaShop/pull/35843/ (in dev branch)
| Type?             | 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | 
| How to test?      | 

### How to test
- Upgrade from 8.1 to 9.0
- Open phpmyadmin
- See that: 
  - A) `meta_keywords` columns is removed from all tables.
  - B) There is `PS_DEFAULT_LANGUAGE_URL_PREFIX` record in `ps_configuration`.
  - C) There is `PS_SEPARATOR_FILE_MANAGER_SQL` record in `ps_configuration`.
- No need to test anything else, not in the scope of this PR

### CODE REVIEW

**Review by M0rgan01 — APPROVED** [review]

**Comment by M0rgan01** [comment]
@Hlavtox PR https://github.com/PrestaShop/autoupgrade/pull/1010 will fix CI

**Comment by Hlavtox** [comment]
@M0rgan01 Ok I will rebase after it's merged. :-)

**Comment by Hlavtox** [comment]
@M0rgan01 Ready to go, tests should be green. :-)

**Review by M0rgan01 — APPROVED** [review]

**Review by ga-devfront — APPROVED** [review]

**Comment by AureRita** [comment]
Hi @Hlavtox 

Your Pr is currently in conflict, maybe because this [PR](https://github.com/PrestaShop/autoupgrade/pull/691) was merged, 
you can see the conflict here : 

image

Waiting for feedback

**Comment by Hlavtox** [comment]
@AureRita You are rebasing to wrong branch, this PR targets 6.x

**Comment by AureRita** [comment]
Yes Indeed, sorry didn't see that this parameter change

**Review by AureRita — APPROVED** [review]
Hi @Hlavtox 

Thank you for your PR, I tested it and it seems to works as you can see : 

[recording(335).webm](https://github.com/user-attachments/assets/38f52aa0-26de-4ce7-94fc-e7d636fadcff)


Tested from : 
8.2 to 9.0.0
8.0.4 to 9.0.0

Because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Pull request merged**
